bears/general/InvalidLinkBear.py

```diff
@@ -42,7 +42,7 @@ def get_status_code(url, timeout):
     def find_links_in_file(file, timeout, ignore_regex):
         ignore_regex = re.compile(ignore_regex)
         regex = re.compile(
-            r'((ftp|http)s?://\S+\.(?:[^\s()\'\"<>|]+|'
+            r'((ftp|http)s?://[^.:\s_/?#[\]@]+\.(?:[^\s()\'\"<>|]+|'
```

Comment:
yeah we looked it up, not sure where it is. In the later part we have added a bit extra (in another commit) to make a heuristic which actually seems to work better then GMails one in terms of finding the URL that the user "means" ;)